project/app/utils/README.md

```diff
@@ -1 +1,3 @@
-The 'google_api.py' file has the code to connect to Google Cloud Vision API and return the transcribed text extracted from a given image
\ No newline at end of file
+#### Overview of `utils` content:
+- `transcription.py`: Utilize's the Google Cloud Vision API and their document_text_detection method to transcribe text from a given image
+- `safe_search.py`: Utilize's the Google Cloud Vision API and their safe_search method to perform moderation of user uploaded illustrations
```

Comment:
You can remove the apostrophe from "Utilizes". Also I'm a fan of formatting anything that could be code-specific, as code. So my preference would be: `document_text_detection` and `safe_search`.